ros_gz_image/src/image_bridge.cpp

```diff
@@ -39,13 +39,13 @@ class Handler
     std::shared_ptr<gz::transport::Node> _gz_node)
   {
     // Get QoS profile from parameter
-    rmw_qos_profile_t qos_profile = rmw_qos_profile_default;
+    rclcpp::QoS qos_profile = rclcpp::QoS(10);
```

Comment:
Do we need to use are we using a magic number (10) here? Is there no equivalant to `rmw_qos_profile_default`?